lms/templates/header/header.html

```diff
@@ -29,6 +31,26 @@
     </%block>
 % endif
 
+<%
+  unsupported_browser_alert_versions = configuration_helpers.get_value('UNSUPPORTED_BROWSER_ALERT_VERSIONS', settings.FEATURES.get('UNSUPPORTED_BROWSER_ALERT_VERSIONS'))
+%>
+% if waffle.switch_is_active('enable_unsupported_browser_alert'):
+  <script>
+  var $buoop = {
+    notify:${unsupported_browser_alert_versions | n, decode.utf8},
+    api:5,
+    reminder:0
+  };
+  function $buo_f(){
+   var e = document.createElement("script");
```

Comment:
Alll variables should be declared at the top of the script, per our Javascript style guides.